Sources/TuistCore/Cache/CacheDirectoriesProvider.swift

```diff
@@ -21,25 +21,21 @@ public final class CacheDirectoriesProvider: CacheDirectoriesProviding {
     }
 
     public func cacheDirectory(for category: CacheCategory) -> AbsolutePath {
-        cacheDirectory.appending(component: category.directoryName)
-    }
-}
-
-extension CacheCategory {
-    var directoryName: String {
-        switch self {
+        switch category {
```

Comment:
There's an opportunity to expose a variable from `CacheCategory` and move the switch-case there:
```swift
enum CacheCategory {
  var directoryName: String {
    ...
  }
}
```

Reply:
That's what I did initially, but given now not all the caches are in the "main cache folder" (e.g. the dependencies cache is local), adding such helper could cause some wrong assumptions (e.g. once you have a directoryName you assume the folder is in `~/.tuist/Cache/<directoryName>`, but it is not always the case)

Reply:
In that case, we are adding an element to `CacheCategory` that shouldn't be there. `CacheCategory` represents the cache directories under the global cache and the `dependencies` category is specific to the dependencies feature. If `CacheCategory` name made you think there's space for `dependencies`, we should rename `CacheCategory` to something more explicit about what the group is about.